ui/src/store/modules/graph.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+
+const mockAPI = process.env.VUE_APP_MOCK_API;
+
+const state = {
+    data: []
+}
+
+const actions = {
+    setAnalytics: ({ commit }) => {
+        axios.get(mockAPI)
+        .then(response => {
+            commit('SET_ANALYTICS_DATA', response.data);
+        })
+    }
+}
+
+const getters = {
+    getGraphData: function(state) {
+        return state;
+    }
```

Comment:
Getter for the entire state is meaningless and you lose the point in using getters this way. Can you make the getter retrieve something more specific? Also getters shouldn't sound like functions.
```js
graphData: function (state) {
  return state.data;
}
```